src/rgw/rgw_iam_policy.h

```diff
@@ -9,7 +9,6 @@
 #include <cstdint>
 #include <iostream>
 #include <string>
-#include <bitset>
```

Comment:
why remove the header for std::bitset?

Reply:
It's included twice.

Reply:
@cbodley 
Because it is a duplicate. The first include is also a `#include bitset` 
code style asks for includes to be aphabetically ordered.